lang/pure/Portfile

```diff
@@ -1,17 +1,17 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem              1.0
-PortGroup               bitbucket 1.0
+PortGroup               github 1.0
 
 # Keep the versions of pure and pure-docs in sync.
-bitbucket.setup         purelang pure-lang 0.66 pure-
-name                    pure
+name                            pure
+github.setup                    agraef pure-lang 0.67 ${name}-
```

Comment:
Could you fix the alignment of these two lines?

Reply:
Done.